content/docs/start/experiments.md

```diff
@@ -4,237 +4,227 @@ title: 'Get Started: Experiments'
-[was created](/doc/start/data-pipelines#dependency-graphs-dags) with this
-`dvc run` command. Notice the argument sent to the `-p` option (short for
-`--params`):
+`dvc exp show` compares any number of experiments in one table:
 
 ```dvc
-$ dvc run -n featurize \
-          -p featurize.max_features,featurize.ngrams \
-          -d src/featurization.py -d data/prepared \
-          -o data/features \
-          python src/featurization.py data/prepared data/features
+$ dvc exp show --no-timestamp --include-params train.n_est,train.min_split
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2
+│ master        │  0.55259 │ 0.91536 │ 50         │ 2
+│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8
+│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64
+│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2
+│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8
+│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64
+│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2
+└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
 ```
 
-</details>
+Each experiment is given an arbitrary name by default (although we can specify
+one with `dvc exp run -n`.) We can see that `exp-98a96` performed best among
+both of our metrics, with 100 estimators and a minimum of 64 samples required to
+split a node.
 
-The `params` section defines the [parameter](/doc/command-reference/params)
-dependencies of the `featurize` stage. By default DVC reads those values
-(`featurize.max_features` and `featurize.ngrams`) from a `params.yaml` file. But
-as with metrics and plots, parameter file names and structure can also be user
-and case defined.
+> See `dvc exp show --help` for more info on its options.
 
-This is how our `params.yaml` file looks like:
+## Promoting experiments
 
-```yaml
-prepare:
-  split: 0.20
-  seed: 20170428
+Now that we know the best parameters, let's keep that experiment and ignore the
+rest.
 
-featurize:
-  max_features: 500
-  ngrams: 1
+`dvc exp apply` checks out the specified experiment to our workspace:
 
-train:
-  seed: 20170428
-  n_estimators: 50
+```dvc
+$ dvc exp apply exp-98a96
+Changes for experiment 'exp-98a96' have been applied to your current workspace.
 ```
 
-## Tuning and running experiments
+<details>
 
-We are definitely not happy with the `AUC` value we got so far! Let's now tune
-and run the new experiment. Edit the `params.yaml` file to use bigrams and
-increase the number of features:
+### 💡 Expand to see what this command does.
 
-```diff
- featurize:
--  max_features: 500
--  ngrams: 1
-+  max_features: 1500
-+  ngrams: 2
-```
+`dvc exp apply` is similar to `dvc checkout` but it works with experiments that
+have not been manually committed to the Git repo. DVC tracks everything in the
```

Comment:
I'm not sure what you mean, so maybe I made a great point by accident? Could you explain more? The typical workflow would likely be to use `exp run` a bunch of times followed by `exp apply` and `git commit`.

Reply:
> > great point. I never thought that you could exp run a few times and git add/commit it yourself
>
> The typical workflow would likely be to use exp run a bunch of times followed by exp apply and git commit.

Exactly. I was assuming that's the only workflow but "have not been manually committed" made me realize users could just use `exp run` (even repeating a past exp instead of `exp apply`) and check the results with `metrics/exp diff/show` (or manually), then commit when they're satisfied (which resets `exp show`).

Anyway. I'm just wondering if we should consider that workflow seriously. And if now, why mention "have not been manually committed"? Minor

Reply:
I think this is poorly worded rather than making a great point. The intention was to convey that `dvc checkout` is to persistent (Git-committed) pipeline iterations what `dvc exp apply` is to ephemeral experiments. 

Maybe I should just drop `that have not been manually committed to the Git repo`?

Reply:
Dropping for now.

Reply:
Sounds good 👍